### DISCUSSION TITLE
Does --taxon-exclude work with taxa above species?

### DISCUSSION THREAD

**Discussion by Benjamin-Lee** [bbuchfink/diamond#455] [Q&A] [answered]
The docs are somewhat unclear on this. If I want to exclude hits from [28384](https://www.ncbi.nlm.nih.gov/taxonomy/?term=28384) (vectors, etc), do I need to pass each taxid individually (like regular BLAST) or will 28384 alone suffice?

**Comment by bbuchfink** [comment, marked as answer]
Higher nodes work as well, you don't need to list everything.
